### DISCUSSION TITLE
How to upgrade from 0.8

### DISCUSSION THREAD

**Discussion by lz1998** [SeaQL/sea-orm#894] [Q&A] [answered]
Do I need to wrap a `CustomJson` to support serde_json::Value?

the trait `From<serde_json::Value>` is not implemented for `sea_orm::Value`

0.8
```rust
pub setting: Json
```

0.9
```rust
pub setting: serde_json::Value
```

**Comment by lz1998** [comment, marked as answer]
I add feature `with-json`, and it works.

**Comment by tyt2y3** [comment]
Thanks. I now added some docs in Changelog. If you come from an older version of sea-orm, you can now remove the `default-features =  false`.
